modules/cluster-logging-updating-logging.adoc

```diff
@@ -120,46 +107,144 @@ oc exec -n openshift-logging -c elasticsearch elasticsearch-cdm-1pbrl44l-1-55b75
 ----
-image: registry.redhat.io/openshift4/ose-logging-fluentd@sha256:20f92b37685bc1003bb490002f7d8a1abee2dd2d157e8532afa3830ce8da3483
-image: registry.redhat.io/openshift4/ose-logging-fluentd@sha256:20f92b37685bc1003bb490002f7d8a1abee2dd2d157e8532afa3830ce8da3483
-image: registry.redhat.io/openshift4/ose-logging-fluentd@sha256:20f92b37685bc1003bb490002f7d8a1abee2dd2d157e8532afa3830ce8da3483
-image: registry.redhat.io/openshift4/ose-logging-fluentd@sha256:20f92b37685bc1003bb490002f7d8a1abee2dd2d157e8532afa3830ce8da3483
-image: registry.redhat.io/openshift4/ose-logging-fluentd@sha256:20f92b37685bc1003bb490002f7d8a1abee2dd2d157e8532afa3830ce8da3483
-image: registry.redhat.io/openshift4/ose-logging-fluentd@sha256:20f92b37685bc1003bb490002f7d8a1abee2dd2d157e8532afa3830ce8da3483
-image: registry.redhat.io/openshift4/ose-logging-fluentd@sha256:20f92b37685bc1003bb490002f7d8a1abee2dd2d157e8532afa3830ce8da3483
-image: registry.redhat.io/openshift4/ose-logging-fluentd@sha256:20f92b37685bc1003bb490002f7d8a1abee2dd2d157e8532afa3830ce8da3483
-image: registry.redhat.io/openshift4/ose-logging-fluentd@sha256:20f92b37685bc1003bb490002f7d8a1abee2dd2d157e8532afa3830ce8da3483
-image: registry.redhat.io/openshift4/ose-logging-fluentd@sha256:20f92b37685bc1003bb490002f7d8a1abee2dd2d157e8532afa3830ce8da3483
-image: registry.redhat.io/openshift4/ose-logging-fluentd@sha256:20f92b37685bc1003bb490002f7d8a1abee2dd2d157e8532afa3830ce8da3483
-image: registry.redhat.io/openshift4/ose-logging-fluentd@sha256:20f92b37685bc1003bb490002f7d8a1abee2dd2d157e8532afa3830ce8da3483
+"containerName": "fluentd-init"
 ----
 
-.. Ensure that the Kibana Pods are using a 4.5 image:
+.. Verify that the log visualizer is updated to 4.5 using the Kibana CRD:
 +
 ----
-$ oc get pod -n openshift-logging --selector logging-infra=kibana -o yaml |grep 'image:'
+$ oc get kibana kibana -o json
 ----
 +
+There is a Kibana pod with the `ready` status:
++
 ----
-image: registry.redhat.io/openshift4/ose-logging-kibana5@sha256:3d657e3b90fae604a8351b1923250f93c04529b36e6ada0aba7c0a038ffef56e
-image: registry.redhat.io/openshift4/ose-oauth-proxy@sha256:5fe478210770b21c1eb26c1570bcbda40bc5a79011580ff5ebd4c701a5b04eb2
-image: registry.redhat.io/openshift4/ose-logging-kibana5@sha256:3d657e3b90fae604a8351b1923250f93c04529b36e6ada0aba7c0a038ffef56e
-image: registry.redhat.io/openshift4/ose-oauth-proxy@sha256:5fe478210770b21c1eb26c1570bcbda40bc5a79011580ff5ebd4c701a5b04eb2
+[
+{
+"clusterCondition": {
+"kibana-5fdd766ffd-nb2jj": [
+{
+"lastTransitionTime": "2020-06-30T14:11:07Z",
+"reason": "ContainerCreating",
+"status": "True",
+"type": ""
+},
+{
+"lastTransitionTime": "2020-06-30T14:11:07Z",
+"reason": "ContainerCreating",
+"status": "True",
+"type": ""
+}
+]
+},
+"deployment": "kibana",
+"pods": {
+"failed": [],
+"notReady": []
+"ready": []
+},
+"replicaSets": [
+"kibana-5fdd766ffd"
+],
+"replicas": 1
+}
+]
 ----
 
-.. Ensure that the Curator CronJob is using a 4.5 image:
+.. Verify the Curator is updated to 4.5:
```

Comment:
Can you double check the formatting here? The output in the preview is showing markup.

Reply:
I fixed that earlier today. The preview must not have updated (yet?).